packages/landing/src/tile.matrix.ts

```diff
@@ -54,6 +55,27 @@ export class TileGrid {
         });
     }
 
+    getStyle(config: MapOptions): Style | string {
+        if (config.imageId === 'topographic') return WindowUrl.toTileUrl(config, MapOptionType.VectorTile);
+        return {
+            version: 8,
+            sources: {
+                'raster-tiles': {
```

Comment:
would be easier if this was not `-` eg `raster` or `basemaps` then you wont have to quote it.